addons/uh60_misc/functions/fnc_perFrame.sqf

```diff
@@ -16,6 +16,10 @@ if (inputAction "headlights" > 0 && !vtx_uh60_misc_btn_hold) then {
     };
 };
 
+if (!vtx_uh60_misc_landingLightOn && !(_vehicle animationPhase "Switch_lights_position" > 0.5) && !(_vehicle animationPhase "Switch_lights_cockpit" > 0.5)) then {
```

Comment:
does this need to run on every frame or can it run once per second

Reply:
Or can we maybe use an eventhandler

Reply:
The logic should be handled in vtx_uh60_misc_fnc_toggleLights, which is run by all the light switches as well as the basegame input.

Reply:
I think i got bogged down in things not working properly and forgot to move it back to toggleLights fnc. 

Resolved & ready to merge